public/content/documentation/web/component/search.md

```diff
@@ -0,0 +1,163 @@
+
+## Condensed
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab: Focus moves visibly to the search text input and search button
+   - Space: Search button is activated
+   - Enter: Search is activated
+
+2. Test mobile screenreader gestures
+   - Swipe: Focus moves to the search text input and search button
+   - Doubletap: Search button is activated
+
+3. Listen to screenreader output on all devices
+
+   - Name: Its purpose is clear
+   - Role: It identifies itself as a search input
+   - Group: The form itself is discoverable with screenreader shortcuts as search landmark
+
+Full information: [https://www.magentaa11y.com/checklist-web/search/](https://www.magentaa11y.com/checklist-web/search/)
+
+## Gherkin
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a header
+
+GIVEN THAT I am on a page with a header landmark
+
+1. Keyboard for mobile & desktop
+
+   - WHEN I use the tab key to move focus to a search input I SEE focus is strongly visually indicated
+   - THEN when I use the tab key to move focus to the search submit button I SEE the button is focused
+   - THEN when I use the enter or spacebar key I SEE the search results are presented
+
+2. Desktop screenreader
+
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND 
+   - I use the tab key to move focus to a search input
+      - I HEAR Its purpose is clear
+      - I HEAR It identifies itself as a search input
+      - I HEAR The form itself is discoverable with screenreader shortcuts as search landmark
+   - THEN when I use the tab key to move focus to the search submit button I HEAR the button is focused
+   - THEN when I use the enter or spacebar key I HEAR the search results are presented
+
+3. Mobile screenreader
+
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+   - I swipe to focus on a search input
+      - I HEAR Its purpose is clear
+      - I HEAR It identifies itself as a search input
+      - I HEAR The form itself is discoverable with screenreader
+
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/search/search](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/search/search)
+
+
+## Videos
+
+### Android Talkback
+
+<video controls>
+  <source src="media/video/web/search/search-android.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+### iOS VoiceOver
+
+<video controls>
+  <source src="media/video/web/search/search-ios.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+## Code examples
+### Use semantic HTML
+
+- This semantic HTML contains all accessibility features by default. 
+- Include a search button
+
+### Search with autocomplete suggestions
+
+- For search autocomplete see [Autocomplete input with listbox](/checklist-web/listbox-autocomplete/)
+
+```html
+<form role="search" action="/search/">
```

Comment:
remove action attribute from here and the example to get the form to work a bit better